### DISCUSSION TITLE
法國原神

### DISCUSSION THREAD

**Discussion by sky-R6** [share121/inter-knot#2426] [其他]
![capsule_616x353](https://github.com/user-attachments/assets/463bfe44-cb42-435d-ace2-160d324ef6d8)

你說得對，但是《虹彩六號：圍攻行動》為一款5V5第一人稱戰術射擊遊戲，遊戲中玩家可以選擇不同特勤幹員為角色，每位角色皆擁有不同的裝備，且可以在槍枝上面裝上不同的小配備，如選擇新進人員作為參與遊戲的特勤幹員，則會擁有數個固定的預設配備可以選擇。遊戲分為攻擊方與防守方，在排名戰模式以及休閒模式下選取角色完畢後隊伍可以以投票的形式選擇要在哪一個地點重生。回合開始後攻擊方可以使用無人機探查地圖不同地方試圖找出防守方的位置，防守方則要破壞無人機、用木板封阻門框或窗戶、加固牆面以避免該牆壁被攻擊方破壞用以侵入目標，防守方也可利用監視器找出敵人位置。遊戲的目標為攻擊方必須攻堅目的地後達成條件，防守方則要防止攻擊方達成勝利條件。攻擊方除了可以破壞牆面之外也可以在建築物外側進行繩降，這些可破壞的表面也可以被防守方拿來偷襲與利用。遊戲中的武器可以擊傷隊友，且角色死亡後直到下一回合前都無法復活。遊戲共有四種完結方式：攻擊方達成條件後勝利、防守方成功守住據點並且撐到時間結束後勝利、其中一隊殲滅敵隊後勝利、兩隊僅存的最後一名隊友剛好同時死亡後平手。 遊戲的破壞系統為其一大特色，遊戲中易碎材質（如木頭、玻璃等）所組成的表面皆可破壞，包含了牆壁、天花板與地板。這些材質除了使用炸藥也可以用槍擊或近戰攻擊破壞，若牆面遭到加固則需要特殊的炸藥方可擊破。

**Comment by M1nam1Yume** [comment]
没想到吧，来这里也不能直播

**Comment by hyperion-netizen** [comment]
谁问你了？(doge

**Comment by lllliufeng** [comment]
You are RIGHT, but first I forgot, then I forgot, finally i forgot, all in all i forgot.

**Comment by Rolesican** [comment]
做1不持久，做0不耐干，做t腱鞘炎，做p夹不住，做s会笑场，做m会还手，做3会被打，还是做彩虹6号的电子宠物爽，每天打彩六困得像阿片里的丈夫，瞎的像厨房里的闺蜜，聋的像茶水间里的同事，冷漠的像电车里的乘客，活的像阿片里的妻子一样，毫无意义，被队友压力了，窝囊得像把领导带回家吃饭的丈夫，好累，累的，像三胎妈妈，哄完孩子睡觉之后被老公打了，还要坐小板凳，在凉水里洗衣服，肚子里还揣着一个，婆婆还在我面前破口大骂，骂完去买菜，结果小三找上门侮辱我的无力感，做0当1当3去4我不如玩彩虹6号刺激一7来打彩6吧，9是放不下10一样的彩虹六号，对不起，老公，这辈子我跟定彩虹六号了
